integration/simple-cache-client.test.ts

```diff
@@ -61,13 +61,13 @@ async function withCache(
 }
 
 describe('SimpleCacheClient.ts Integration Tests', () => {
-  it('should create and delete a cache', async () => {
+  it('should create and delete a cache, set and get a value', async () => {
     const cacheName = v4();
     const momento = new SimpleCacheClient(AUTH_TOKEN, 1111);
     await withCache(momento, cacheName, async () => {
       await momento.set(cacheName, 'key', 'value');
       const res = await momento.get(cacheName, 'key');
-      expect(res instanceof CacheGet.Error).toEqual(false);
+      expect(res instanceof CacheGet.Hit).toEqual(true);
```

Comment:
Note there is [`toBeInstanceOf`](https://jestjs.io/docs/expect#tobeinstanceofclass). More concise, easier to read.

```suggestion
      expect(res).toBeInstanceOf(CacheGet.Hit)
```

Reply:
Nice, thanks.